Jenkinsfile

```diff
@@ -114,6 +114,15 @@ pipeline
         always
         {
             junit 'reports/*.xml'
+            // Coverage report
```

Comment:
Can you please just upload it as an artifact? The eclipse foundation do not accept it

Reply:
zip first .
archiveArtifacts artifacts: 'CoverageReport.zip', followSymlinks: false